packages/core/src/modules/ledger/IndyPool.ts

```diff
@@ -9,11 +9,17 @@ import { isIndyError } from '../../utils/indyError'
 import { LedgerError } from './error/LedgerError'
 import { isLedgerRejectResponse, isLedgerReqnackResponse } from './ledgerUtil'
 
+export interface TransactionAuthorAgreement {
+  version: `${number}.${number}` | `${number}`
```

Comment:
suggestion in issue description was ``version: `${number}.${number}` `` only. Maybe I'm missing sth but version I found was of format `` `${number}` `` so I added that as alternative type/format